src/main/java/edu/ucsb/cs56/ucsb_courses_search/model/search/SearchByInstructorMultiQuarter.java

```diff
@@ -24,6 +24,10 @@ public int getEndQ() {
     }
 
     public void setEndQ(int endQ) {
+       /* //or message to say end must be later than 
```

Comment:
If it is commented out, it might be better to remove it.

Reply:
okay makes sense! went ahead and deleted these lines